java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -1892,6 +1893,29 @@ void testsubstring() {
     }
   }
 
+  @Test
+  void stringSplit() {
+    try (ColumnVector v = ColumnVector.fromStrings("Héllo there", "thésé", null, "", "ARé some", "test strings");
+         Table expected = new Table.TestBuilder().column("Héllo", "thésé", null, "", "ARé", "test")
+         .column("there", null, null, null, "some", "strings")
+         .build();
+         Scalar pattern = Scalar.fromString(" ");
+         Table result = v.stringSplit(pattern)) {
+      assertTablesAreEqual(expected, result);
+    }
+  }
+
+  @Test
+  void stringSplitWhiteSpace() {
+    try (ColumnVector v = ColumnVector.fromStrings("Héllo thesé", null, "are\tsome", "tést\nString", " ");
+         Table expected = new Table.TestBuilder().column("Héllo", null, "are", "tést", null)
+         .column("thesé", null, "some", "String", null)
+         .build();
+         Table result = v.stringSplit()) {
+      assertTablesAreEqual(expected, result);
+    }
+  }
+
```

Comment:
Can we get some negative test cases to show that we return errors when expected?